modules/thermal_hydraulics/test/tests/components/volume_junction_1phase/gold/junction_with_calorifically_imperfect_gas_out.csv

```diff
@@ -0,0 +1,2 @@
+time,net_mass_flow_rate_domain,net_mass_flow_rate_volume_junction
+10,-0.043470721613488,-5.1481353452232e-05
```

Comment:
These values should be 0

Reply:
I reduced the end time step to make the test faster (the test is not actually testing balance, but that we can run with both `CaloricallyImperfectGas` and `VolumeJunction1Phase`).

So you just want me to put a larger end time (i.e. larger runtime)?

Reply:
If you use my suggestion for the flow area, use the component `InletVelocityTemperature1Phase` for your inlet pipe. Then you can use that prescribed velocity as your initial condition. It should converge much faster.